src/cljs/org/broadinstitute/firecloud_ui/page/workspaces.cljs

```diff
@@ -125,33 +125,79 @@
         [:div {:style {:clear "both"}}]])}))
 
 
+(defn- modal-background [state]
+  {:backgroundColor "rgba(82, 129, 197, 0.4)"
+   :display (if (:overlay-shown? @state) "block" "none")
+   :overflowX "hidden" :overflowY "scroll"
+   :position "fixed" :zIndex 9999
+   :top 0 :right 0 :bottom 0 :left 0
+   :textAlign "left"})
+
+(def modal-content
+  {:transform "translate(-50%, 0px)"
+   :backgroundColor "#F4F4F4"
+   :position "relative" :marginBottom 60
+   :top 60 :left "50%" :width 500})
+
+(def form-label
+  {:marginBottom "0.16667em" :fontSize 14})
```

Comment:
Consider using a percent value for font sizing (everywhere). I know the designs use pixel values, but percentages make it easier to adjust things for accessibility and mobile.